assets/js/membraneWebRTC.ts

```diff
@@ -662,14 +681,17 @@ export class MembraneWebRTC {
     return trackIdToMetadata;
   };
 
+  private checkIfTrackBelongToPeer = (trackId: string, peer: Peer) =>
+    Array.from(peer.trackIdToMetadata.keys()).some((track) => trackId.startsWith(track));
```

Comment:
I think we can get rid of this function and do something like
```js
peer.trackIdToMetadata[trackId] != undefined
```
Take a look at https://stackoverflow.com/questions/1098040/checking-if-a-key-exists-in-a-javascript-object

Reply:
This won't work, because for some reason, which I don't know simulcast tracks which peer add will call `this.connection!.ontrack` callback. (This callback is used only for tracks that came to the browser from the engine). Also peer doesn't have these tracks in TrackIdToMetadata. Because I was thinking that all simulcast tracks should share metadata, but now I am not sure about this.